controllers/grid/articleGalleys/form/ArticleGalleyForm.php

```diff
@@ -25,34 +25,42 @@
 
 class ArticleGalleyForm extends Form
 {
-    /** @var Submission */
+    /** @var \Submission */
     public $_submission = null;
 
-    /** @var Publication */
+    /** @var \Publication */
```

Comment:
Similar comment as above about the `use` statements.

Reply:
Fixed